internal/declarative/v2/renderer_raw.go

```diff
@@ -112,80 +41,6 @@ func (r *RawRenderer) Render(_ context.Context, obj Object) ([]byte, error) {
 	return manifest, nil
 }
 
-func (r *RawRenderer) RemovePrerequisites(ctx context.Context, obj Object) error {
-	crdLength := len(r.crds)
-	if crdLength == 0 {
-		return nil
-	}
-	status := obj.GetStatus()
-	if err := r.uninstallCRDs(ctx); err != nil {
-		r.Event(obj, "Warning", "CRDsUninstallation", err.Error())
-		obj.SetStatus(status.WithState(StateError).WithErr(err))
-		return err
-	}
-	return nil
-}
-
-func (r *RawRenderer) getCRDs(rawManifestReader io.Reader) error {
-	r.crds = make([]*unstructured.Unstructured, 0)
-	decoder := yaml.NewYAMLOrJSONDecoder(rawManifestReader, defaultBufferSize)
-	for {
-		resource := &unstructured.Unstructured{}
-		err := decoder.Decode(resource)
-		if err != nil && !errors.Is(err, io.EOF) {
-			return err
-		}
-		if errors.Is(err, io.EOF) {
-			break
-		}
-		crdGVK := apiextensionsv1.SchemeGroupVersion.WithKind("CustomResourceDefinition")
-		if resource.GroupVersionKind() == crdGVK {
-			r.crds = append(r.crds, resource)
-			continue
-		}
-	}
-	return nil
-}
-
-func (r *RawRenderer) installCRDs(ctx context.Context) error {
```

Comment:
Is it really obsolete?

Reply:
this renderer_raw just need to support `Render`, there is no need to implement EnsurePrerequisites, RemovePrerequisites, we previously already finish the implementation of all three rendering mode. And this additional is added by @khlifi411 since his previous implementation which is not needed.

Please check my comments [here](https://github.com/kyma-project/lifecycle-manager/pull/469#discussion_r1171675364)
What I did here is bring it back to the previous working version.

Reply:
:+1: